elemica-suggest.coffee

```diff
@@ -180,6 +185,7 @@ elemicaSuggest 0.8.2-SNAPSHOT
             event.preventDefault()
           else if event.keyCode == TAB && isSelectingSuggestion()
             selectHighlighted(this)
+            removeSuggestions(this)
```

Comment:
Previously, `selectHighlighted` was triggering `blur` event that was causing `removeSuggestions` to be executed (see `blur` event handler). Because now I'm calling `selectHighlighted` from `blur` handler, it can no longer trigger `blur` event - we would have an endless loop. That's why `removeSuggestions` are called explicitly here.

Probably, we'd be able to move `removeSuggestions` to `selectHighlighted` but I have some doubts if it would be a good move. `selectHighlighted` is just selecting, it shouldn't remove anything.